From e464db226f8014a4b199a6d6b2b177057af6a248 Mon Sep 17 00:00:00 2001
From: Mandriva Contributor <contributor@mandriva.org>
Date: Wed, 24 Jun 2015 09:24:03 +0100
Subject: [PATCH 901/907] leveldb: Change runlevels dir to suit Mandriva.

Authorship not fully looked up - see Mandriva package SVN for history
---
 leveldb.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/leveldb.h b/leveldb.h
index 6799264..f9a52b1 100644
--- a/leveldb.h
+++ b/leveldb.h
@@ -16,7 +16,7 @@
 #ifndef H_LEVELDB
 #define H_LEVELDB
 
-#define RUNLEVELS "/etc"
+#define RUNLEVELS "/etc/rc.d"
 #define XINETDDIR "/etc/xinetd.d"
 
 #include <glob.h>
-- 
2.3.2

